flujo/application/core/agents/agent_plugin_runner.py

```
@@ -0,0 +1,388 @@
+                resources_closed=False,
+                state=state,
+                attempt_exception=None,
+            )
+
+        def _safe_step_name(step_obj: object) -> str:
+            safe_fn = getattr(core, "_safe_step_name", None)
+            if callable(safe_fn):
+                try:
+                    return str(safe_fn(step_obj))
+                except Exception:
+                    pass
```

Ruff (0.14.8)

`try`-`except`-`pass` detected, consider logging the exception

(S110)